package_mac.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source ./harness-util/package-util.sh
+create_dist_pkg "mac" "x86-64" "macosx_10_9_x86_64"
```

Comment:
I propose renaming `package_mac.sh` and `package_win.sh` to `package_macos.sh` and `package_windows.sh` respectively because this is now these OSes are actually named. Correspondingly, these scripts will need to pass `"macos"` and `"windows"` as the first argument to the `create_dist_pkg` function.